### DISCUSSION TITLE
Requirements / limitations for logo.svg?

### DISCUSSION THREAD

**Discussion by dimatx** [timlrx/tailwind-nextjs-starter-blog#862] [General]
Wondering if there are specifications that must be followed for the SVG logo.

I tried swapping it out with a few SVGs to test things, for example this one: https://www.svgviewer.dev/s/208278/logo

And when the site builds, nothing seems to show up in the place of the logo.

![image](https://github.com/timlrx/tailwind-nextjs-starter-blog/assets/24803941/d997c9ac-8e59-462e-ad9f-f219d7b97e1a)

**Comment by ninjoala** [comment]
Any luck on this? I'm having the same issue.

**Reply by dimatx** [reply]
Not yet. Unsure what may be the cause.
